core/kernel/autoload_spec.rb

```diff
@@ -42,6 +42,10 @@ def check_autoload(const)
     KSAutoloadB.loaded.should == :ksautoload_b
   end
 
+  it "loads the file at a top level" do
+    ruby_exe(fixture(__FILE__, "autoload_top.rb")).should.strip == "ksautoload_b"
```

Comment:
Don't the autoload's at lines 8-10 and the spec just above already tests that?

Reply:
yes, I would expect, that "loads the file when the constant is accessed" fails, but it doesn't for some reason.